src/_try-dart-examples.md

```diff
@@ -180,11 +180,11 @@ void main() {
 ### Compute π
 
 {:#try-dart-compute-pi}
-<?code-excerpt "misc/lib/pi_monte_carlo.dart (try-dart)" plaster="none" remove="/output|window/" replace="/numIterations/100/g; /\n *\/\/!.*\n/\n/g; / *\/\/!.*\n/\n/g"?>
+<?code-excerpt "../null_safety_examples/misc/lib/pi/pi_monte_carlo.dart (try-dart)" plaster="none" remove="/output|window/" replace="/numIterations/100/g; /\n *\/\/!.*\n/\n/g; / *\/\/!.*\n/\n/g"?>
 ```dart
 import 'dart:math' show Random;
 
-main() async {
+Future<void> main() async {
```

Comment:
@parlough, I see `void main()` in the "Compute Pi" example on https://dart.dev/#try-dart. Whyyyyy?

(Clearly I don't understand how this page is composed.)

Reply:
We used to collect the examples from the page so examples were verified and tested but at some point this functionality was removed and they are now hardcoded into the DartPad picker code. I have created https://github.com/dart-lang/site-www/issues/3305 to track testing these again so everything will match.